style.css

```diff
@@ -24,6 +24,10 @@ body {
   background-color: #fff;
 }
```

Comment:
- Kindly, Great Job Implementing the previous reviewer suggestions 👌👍, however it needs some change, that is by default some HTML tags have their own style so you have to override them, so it best practice to use 👌👍
```
* {
margin: 0;
padding: 0;
}
```
 you are having the issue below the contact section because of `body` has default `margin:8px` 👌👍